packages/commons/src/auth/authenticationMiddleware.ts

```diff
@@ -20,80 +22,34 @@ export const authenticationMiddleware: (
 ) => ZodiosRouterContextRequestHandler<ExpressContext> =
   (config: JWTConfig) =>
   async (req, res, next): Promise<unknown> => {
-    const jwksClients = getJwksClients(config);
-
-    const addCtxAuthData = async (
```

Comment:
Logic is simpler now, no more need for this sub-function, I moved the code directly into the try catch